### PR TITLE
Update justify-content

### PR DESCRIPTION
**Pull request by lb-fes**
- Add Try it
- Justify structure
- update some translation

### CODE REVIEW

File: files/zh-cn/web/css/justify-content/index.md [hunk 1]
```diff
@@ -1,11 +1,21 @@
 ---
 title: justify-content
 slug: Web/CSS/justify-content
+page-type: css-property
+browser-compat: css.properties.justify-content
```
Comment (yin1999) [inline files/zh-cn/web/css/justify-content/index.md:5]: please remove this part, see: https://github.com/mdn/translated-content/blob/main/docs/zh-cn/translation-guide.md#%E5%85%83%E6%95%B0%E6%8D%AE

**Comment by lb-fes** [comment]
ok, my work is over.

File: files/zh-cn/web/css/justify-content/index.md [hunk 2]
```diff
@@ -5,7 +5,15 @@ slug: Web/CSS/justify-content
 
 {{CSSRef}}
 
-[CSS](/zh-CN/docs/CSS) **`justify-content`** 属性定义了浏览器之间，如何分配顺着弹性容器主轴 (或者网格行轴) 的元素之间及其周围的空间。
+[CSS](/zh-CN/docs/CSS) **`justify-content`** 属性定义浏览器如何沿着弹性容器的{{Glossary("主轴")}}和网格容器的内联轴分配内容项之间和周围的空间。
```
Comment (yin1999) [inline files/zh-cn/web/css/justify-content/index.md:8]:
```suggestion
[CSS](/zh-CN/docs/Web/CSS) **`justify-content`** 属性定义浏览器如何沿着弹性容器的{{Glossary("Main Axis", "主轴")}}和网格容器的内联轴分配内容项之间和周围的空间。
```

File: files/zh-cn/web/css/justify-content/index.md [hunk 3]
```diff
@@ -5,7 +5,15 @@ slug: Web/CSS/justify-content
...
+
+下面的互动示例演示了使用网格布局的一些值。
+
+{{EmbedInteractiveExample("pages/css/justify-content.html")}}
+
+当 length 属性和自动外边距属性（margin: auto）生效之后，对齐已经完成了。也就是说，如果存在至少一个弹性元素，而且这个元素的 {{cssxref("flex-grow")}} 属性不等于 0，那么对齐方式不会生效，就像没有多余空间的情况。
```
Comment (yin1999) [inline files/zh-cn/web/css/justify-content/index.md:14]:
```suggestion
当 length 属性和自动外边距属性生效之后，对齐已经完成了。也就是说，如果[弹性布局](/zh-CN/docs/Web/CSS/CSS_Flexible_Box_Layout)中存在至少一个弹性元素，而且这个元素的 {{cssxref("flex-grow")}} 属性不等于 `0`，那么对齐方式不会生效，就像没有多余空间的情况。
```

File: files/zh-cn/web/css/justify-content/index.md [hunk 4]
```diff
@@ -5,7 +5,15 @@ slug: Web/CSS/justify-content
 
 {{CSSRef}}
 
-[CSS](/zh-CN/docs/CSS) **`justify-content`** 属性定义了浏览器之间，如何分配顺着弹性容器主轴 (或者网格行轴) 的元素之间及其周围的空间。
+[CSS](/zh-CN/docs/CSS) **`justify-content`** 属性定义浏览器如何沿着弹性容器的{{Glossary("主轴")}}和网格容器的内联轴分配内容项之间和周围的空间。
+
+下面的互动示例演示了使用网格布局的一些值。
```
Comment (yin1999) [inline files/zh-cn/web/css/justify-content/index.md:10]:
```suggestion
下面的交互示例演示了使用网格布局的一些值。
```

File: files/zh-cn/web/css/justify-content/index.md [hunk 5]
```diff
@@ -5,7 +5,15 @@ slug: Web/CSS/justify-content
...
+
+{{EmbedInteractiveExample("pages/css/justify-content.html")}}
+
+当 length 属性和自动外边距属性（margin: auto）生效之后，对齐已经完成了。也就是说，如果存在至少一个弹性元素，而且这个元素的 {{cssxref("flex-grow")}} 属性不等于 0，那么对齐方式不会生效，就像没有多余空间的情况。
+
+## Syntax
```
Comment (yin1999) [inline files/zh-cn/web/css/justify-content/index.md:16]:
```suggestion
## 语法
```

File: files/zh-cn/web/css/justify-content/index.md [hunk 6]
```diff
@@ -42,46 +50,46 @@ justify-content: initial;
 justify-content: unset;
 ```
 
-当 length 属性和自动外边距属性（margin: auto）生效之后，对齐已经完成了。也就是说，如果存在至少一个弹性元素，而且这个元素的 {{cssxref("flex-grow")}} 属性不等于 0，那么对齐方式不会生效，就像没有多余空间的情况。
-
-{{cssinfo}}
-
-可以参考 [使用 CSS 弹性框](/zh-CN/docs/Web/CSS/CSS_flexible_box_layout/Using_CSS_flexible_boxes)获取更多信息。
-
 ## 语法
```
Comment (yin1999) [inline files/zh-cn/web/css/justify-content/index.md:54]:
```suggestion
```

File: files/zh-cn/web/css/justify-content/index.md [hunk 7]
```diff
@@ -42,46 +50,46 @@ justify-content: initial;
...
 - `start`
   - : 从行首开始排列。每行第一个元素与行首对齐，同时所有后续的元素与前一个对齐。
+- `end`
+  - : 从行尾开始排列。每行最后一个元素与行尾对齐，同时所有前面的元素与后一个对齐。
 - `flex-start`
-  - : 从行首开始排列。每行第一个弹性元素与行首对齐，同时所有后续的弹性元素与前一个对齐。
+  - : 项目紧密地排列在弹性容器的主轴起始边缘。仅应用于弹性布局的项目。对于不是弹性容器的子项，此值将被视为 `start` 。
 - `flex-end`
-  - : 从行尾开始排列。每行最后一个弹性元素与行尾对齐，其他元素将与后一个对齐。
+  - : 项目紧密地排列在弹性容器的主轴结束边缘。仅应用于弹性布局的项目。对于不是弹性容器的子项，此值将被视为 `end` 。
```
Comment (yin1999) [inline files/zh-cn/web/css/justify-content/index.md:64]: coud you translate "item" into "元素", as the context does

File: files/zh-cn/web/css/justify-content/index.md [hunk 8]
```diff
@@ -42,46 +50,46 @@ justify-content: initial;
...
 - `space-between`
   - : 在每行上均匀分配弹性元素。相邻元素间距离相同。每行第一个元素与行首对齐，每行最后一个元素与行尾对齐。
 - `space-around`
   - : 在每行上均匀分配弹性元素。相邻元素间距离相同。每行第一个元素到行首的距离和每行最后一个元素到行尾的距离将会是相邻元素之间距离的一半。
 - `space-evenly`
   - : flex 项都沿着主轴均匀分布在指定的对齐容器中。相邻 flex 项之间的间距，主轴起始位置到第一个 flex 项的间距，主轴结束位置到最后一个 flex 项的间距，都完全一样。
 - `stretch`
-  - : If the combined size of the items is less than the size of the alignment container, any `auto`-sized items have their size increased equally (not proportionally), while still respecting the constraints imposed by {{cssxref("max-height")}}/{{cssxref("max-width")}} (or equivalent functionality), so that the combined size exactly fills the alignment container along the main axis.
+
+  - : 如果项目沿主轴的组合尺寸小于对齐容器的尺寸，任何 auto-sized 项目都会等比例地增加其尺寸（而不是按比例增加），同时仍然遵守由 {{cssxref("max-height")}}/{{cssxref("max-width")}}（或相应功能）施加的约束, 以便沿主轴完全填充对齐容器的组合尺寸。
```
Comment (yin1999) [inline files/zh-cn/web/css/justify-content/index.md:79]: same

File: files/zh-cn/web/css/justify-content/index.md [hunk 9]
```diff
@@ -42,46 +50,46 @@ justify-content: initial;
...
+
+    > **注意:** 虽然在弹性盒子（flexbox）中支持 `stretch` 属性，但当应用于弹性盒子时，由于拉伸是由 {{CSSXref("flex")}} 属性控制的, 所以 `stretch` 的行为与 `start` 相同。
```
Comment (yin1999) [inline files/zh-cn/web/css/justify-content/index.md:81]: Please check: https://developer.mozilla.org/zh-CN/docs/MDN/Writing_guidelines/Howto/Markdown_in_MDN#%E5%A4%87%E6%B3%A8%E3%80%81%E8%AD%A6%E5%91%8A%E5%92%8C%E6%A0%87%E6%B3%A8

```suggestion
    > **备注：** 虽然在弹性盒子支持 `stretch` 属性，但当应用于弹性盒子时，由于拉伸是由 {{CSSXref("flex")}} 属性控制的，所以 `stretch` 的行为与 `start` 相同。
```

File: files/zh-cn/web/css/justify-content/index.md [hunk 10]
```diff
@@ -42,46 +50,46 @@ justify-content: initial;
...
+
 - `safe`
-  - : 与对齐关键字一起使用，如果选定的关键字会导致元素溢出容器造成数据丢失，那么将会使用 `start` 代替它。
+  - : 如果项目溢出对齐容器，则项目将按照对齐模式为 `start` 进行对齐。所期望的对齐将不会被实现。
 - `unsafe`
-  - : Regardless of the relative sizes of the item and alignment container, the given alignment value is honored.
+  - : 即使项目溢出对齐容器，也会实现所需的对齐方式。与 `safe` 不同，`safe` 会忽略所需的对齐方式以防止溢出。
```
Comment (yin1999) [inline files/zh-cn/web/css/justify-content/index.md:86]: same

File: files/zh-cn/web/css/justify-content/index.md [hunk 11]
```diff
@@ -42,46 +50,46 @@ justify-content: initial;
...
 
 ### 值
 
 - `start`
   - : 从行首开始排列。每行第一个元素与行首对齐，同时所有后续的元素与前一个对齐。
+- `end`
+  - : 从行尾开始排列。每行最后一个元素与行尾对齐，同时所有前面的元素与后一个对齐。
 - `flex-start`
-  - : 从行首开始排列。每行第一个弹性元素与行首对齐，同时所有后续的弹性元素与前一个对齐。
+  - : 项目紧密地排列在弹性容器的主轴起始边缘。仅应用于弹性布局的项目。对于不是弹性容器的子项，此值将被视为 `start` 。
```
Comment (lb-fes) [inline files/zh-cn/web/css/justify-content/index.md:62]:
```suggestion
  - : 元素紧密地排列在弹性容器的主轴起始边缘。仅应用于弹性布局的项目。对于不是弹性容器里的元素，此值将被视为 `start` 。
```

File: files/zh-cn/web/css/justify-content/index.md [hunk 12]
```diff
@@ -42,46 +50,46 @@ justify-content: initial;
...
 - `start`
   - : 从行首开始排列。每行第一个元素与行首对齐，同时所有后续的元素与前一个对齐。
+- `end`
+  - : 从行尾开始排列。每行最后一个元素与行尾对齐，同时所有前面的元素与后一个对齐。
 - `flex-start`
-  - : 从行首开始排列。每行第一个弹性元素与行首对齐，同时所有后续的弹性元素与前一个对齐。
+  - : 元素紧密地排列在弹性容器的主轴起始边缘。仅应用于弹性布局的项目。对于不是弹性容器里的元素，此值将被视为 `start` 。
 - `flex-end`
-  - : 从行尾开始排列。每行最后一个弹性元素与行尾对齐，其他元素将与后一个对齐。
+  - : 项目紧密地排列在弹性容器的主轴结束边缘。仅应用于弹性布局的项目。对于不是弹性容器的子项，此值将被视为 `end` 。
```
Comment (lb-fes) [inline files/zh-cn/web/css/justify-content/index.md:64]:
```suggestion
  - : 元素紧密地排列在弹性容器的主轴结束边缘。仅应用于弹性布局的元素。对于不是弹性容器里的元素，此值将被视为 `end` 。
```

File: files/zh-cn/web/css/justify-content/index.md [hunk 13]
```diff
@@ -42,46 +50,46 @@ justify-content: initial;
...
 - `space-between`
   - : 在每行上均匀分配弹性元素。相邻元素间距离相同。每行第一个元素与行首对齐，每行最后一个元素与行尾对齐。
 - `space-around`
   - : 在每行上均匀分配弹性元素。相邻元素间距离相同。每行第一个元素到行首的距离和每行最后一个元素到行尾的距离将会是相邻元素之间距离的一半。
 - `space-evenly`
   - : flex 项都沿着主轴均匀分布在指定的对齐容器中。相邻 flex 项之间的间距，主轴起始位置到第一个 flex 项的间距，主轴结束位置到最后一个 flex 项的间距，都完全一样。
 - `stretch`
-  - : If the combined size of the items is less than the size of the alignment container, any `auto`-sized items have their size increased equally (not proportionally), while still respecting the constraints imposed by {{cssxref("max-height")}}/{{cssxref("max-width")}} (or equivalent functionality), so that the combined size exactly fills the alignment container along the main axis.
+
+  - : 如果项目沿主轴的组合尺寸小于对齐容器的尺寸，任何 auto-sized 项目都会等比例地增加其尺寸（而不是按比例增加），同时仍然遵守由 {{cssxref("max-height")}}/{{cssxref("max-width")}}（或相应功能）施加的约束, 以便沿主轴完全填充对齐容器的组合尺寸。
```
Comment (lb-fes) [inline files/zh-cn/web/css/justify-content/index.md:79]:
```suggestion
  - : 如果元素沿主轴的组合尺寸小于对齐容器的尺寸，任何 auto-sized 元素都会等比例地增加其尺寸（而不是按比例增加），同时仍然遵守由 {{cssxref("max-height")}}/{{cssxref("max-width")}}（或相应功能）施加的约束, 以便沿主轴完全填充对齐容器的组合尺寸。
```

File: files/zh-cn/web/css/justify-content/index.md [hunk 14]
```diff
@@ -42,46 +50,46 @@ justify-content: initial;
...
 - `stretch`
-  - : If the combined size of the items is less than the size of the alignment container, any `auto`-sized items have their size increased equally (not proportionally), while still respecting the constraints imposed by {{cssxref("max-height")}}/{{cssxref("max-width")}} (or equivalent functionality), so that the combined size exactly fills the alignment container along the main axis.
+
+  - : 如果元素沿主轴的组合尺寸小于对齐容器的尺寸，任何 auto-sized 元素都会等比例地增加其尺寸（而不是按比例增加），同时仍然遵守由 {{cssxref("max-height")}}/{{cssxref("max-width")}}（或相应功能）施加的约束, 以便沿主轴完全填充对齐容器的组合尺寸。
+
+    > **备注：** 虽然在弹性盒子支持 `stretch` 属性，但当应用于弹性盒子时，由于拉伸是由 {{CSSXref("flex")}} 属性控制的，所以 `stretch` 的行为与 `start` 相同。
+
 - `safe`
-  - : 与对齐关键字一起使用，如果选定的关键字会导致元素溢出容器造成数据丢失，那么将会使用 `start` 代替它。
+  - : 如果项目溢出对齐容器，则项目将按照对齐模式为 `start` 进行对齐。所期望的对齐将不会被实现。
```
Comment (lb-fes) [inline files/zh-cn/web/css/justify-content/index.md:84]:
```suggestion
  - : 如果元素溢出对齐容器，则元素将按照对齐模式为 `start` 进行对齐。所期望的对齐将不会被实现。
```

File: files/zh-cn/web/css/justify-content/index.md [hunk 15]
```diff
@@ -42,46 +50,46 @@ justify-content: initial;
...
+  - : 如果元素沿主轴的组合尺寸小于对齐容器的尺寸，任何 auto-sized 元素都会等比例地增加其尺寸（而不是按比例增加），同时仍然遵守由 {{cssxref("max-height")}}/{{cssxref("max-width")}}（或相应功能）施加的约束, 以便沿主轴完全填充对齐容器的组合尺寸。
+
+    > **备注：** 虽然在弹性盒子支持 `stretch` 属性，但当应用于弹性盒子时，由于拉伸是由 {{CSSXref("flex")}} 属性控制的，所以 `stretch` 的行为与 `start` 相同。
+
 - `safe`
-  - : 与对齐关键字一起使用，如果选定的关键字会导致元素溢出容器造成数据丢失，那么将会使用 `start` 代替它。
+  - : 如果元素溢出对齐容器，则元素将按照对齐模式为 `start` 进行对齐。所期望的对齐将不会被实现。
 - `unsafe`
-  - : Regardless of the relative sizes of the item and alignment container, the given alignment value is honored.
+  - : 即使项目溢出对齐容器，也会实现所需的对齐方式。与 `safe` 不同，`safe` 会忽略所需的对齐方式以防止溢出。
```
Comment (lb-fes) [inline files/zh-cn/web/css/justify-content/index.md:86]:
```suggestion
  - : 即使元素溢出对齐容器，也会实现所需的对齐方式。与 `safe` 不同，`safe` 会忽略所需的对齐方式以防止溢出。
```

**Comment by lb-fes** [comment]
Modifications completed

File: files/zh-cn/web/css/justify-content/index.md [hunk 16]
```diff
@@ -5,7 +5,15 @@ slug: Web/CSS/justify-content
...
 {{CSSRef}}
 
-[CSS](/zh-CN/docs/CSS) **`justify-content`** 属性定义了浏览器之间，如何分配顺着弹性容器主轴 (或者网格行轴) 的元素之间及其周围的空间。
+[CSS](/zh-CN/docs/Web/CSS) **`justify-content`** 属性定义浏览器如何沿着弹性容器的{{Glossary("Main Axis", "主轴")}}和网格容器的内联轴分配内容项之间和周围的空间。
+
+下面的交互示例演示了使用网格布局的一些值。
+
+{{EmbedInteractiveExample("pages/css/justify-content.html")}}
+
+当 length 属性和自动外边距属性生效之后，对齐已经完成了。也就是说，如果[弹性布局](/zh-CN/docs/Web/CSS/CSS_Flexible_Box_Layout)中存在至少一个弹性元素，而且这个元素的 {{cssxref("flex-grow")}} 属性不等于 `0`，那么对齐方式不会生效，就像没有多余空间的情况。
```
Comment (yin1999) [inline files/zh-cn/web/css/justify-content/index.md:14]:
```suggestion
当 length 属性和自动外边距属性生效之后，对齐已经完成了。也就是说，如果[弹性布局](/zh-CN/docs/Web/CSS/CSS_flexible_box_layout)中存在至少一个弹性元素，而且这个元素的 {{cssxref("flex-grow")}} 属性不等于 `0`，那么对齐方式不会生效，就像没有多余空间的情况。
```

File: files/zh-cn/web/css/justify-content/index.md [hunk 17]
```diff
@@ -5,7 +5,15 @@ slug: Web/CSS/justify-content
 
 {{CSSRef}}
 
-[CSS](/zh-CN/docs/CSS) **`justify-content`** 属性定义了浏览器之间，如何分配顺着弹性容器主轴 (或者网格行轴) 的元素之间及其周围的空间。
+[CSS](/zh-CN/docs/Web/CSS) **`justify-content`** 属性定义浏览器如何沿着弹性容器的{{Glossary("Main Axis", "主轴")}}和网格容器的内联轴分配内容项之间和周围的空间。
```
Comment (yin1999) [inline files/zh-cn/web/css/justify-content/index.md:8]:
```suggestion
[CSS](/zh-CN/docs/Web/CSS) **`justify-content`** 属性定义浏览器如何沿着弹性容器的{{Glossary("Main Axis", "主轴")}}和网格容器的行向轴分配内容元素之间和周围的空间。
```

File: files/zh-cn/web/css/justify-content/index.md [hunk 18]
```diff
@@ -42,46 +50,44 @@ justify-content: initial;
...
+  - : 元素紧密地排列在弹性容器的主轴起始边缘。仅应用于弹性布局的项目。对于不是弹性容器里的元素，此值将被视为 `start` 。
 - `flex-end`
-  - : 从行尾开始排列。每行最后一个弹性元素与行尾对齐，其他元素将与后一个对齐。
+  - : 元素紧密地排列在弹性容器的主轴结束边缘。仅应用于弹性布局的元素。对于不是弹性容器里的元素，此值将被视为 `end` 。
 - `center`
   - : 伸缩元素向每行中点排列。每行第一个元素到行首的距离将与每行最后一个元素到行尾的距离相同。
 - `left`
   - : 伸缩元素一个挨一个在对齐容器得左边缘，如果属性的轴与内联轴不平行，则`left`的行为类似于`start`。
 - `right`
   - : 元素以容器右边缘为基准，一个挨着一个对齐，如果属性轴与内联轴不平行，则`right`的行为类似于`end`。
```
Comment (yin1999) [inline files/zh-cn/web/css/justify-content/index.md:68]:
```suggestion
  - : 伸缩元素一个挨一个在对齐容器得左边缘，如果属性的轴与内联轴不平行，则 `left` 的行为类似于 `start`。
- `right`
  - : 元素以容器右边缘为基准，一个挨着一个对齐，如果属性轴与内联轴不平行，则 `right` 的行为类似于 `end`。
```

File: files/zh-cn/web/css/justify-content/index.md [hunk 19]
```diff
@@ -42,46 +50,44 @@ justify-content: initial;
...
-
 ### 值
 
 - `start`
   - : 从行首开始排列。每行第一个元素与行首对齐，同时所有后续的元素与前一个对齐。
+- `end`
+  - : 从行尾开始排列。每行最后一个元素与行尾对齐，同时所有前面的元素与后一个对齐。
 - `flex-start`
-  - : 从行首开始排列。每行第一个弹性元素与行首对齐，同时所有后续的弹性元素与前一个对齐。
+  - : 元素紧密地排列在弹性容器的主轴起始边缘。仅应用于弹性布局的项目。对于不是弹性容器里的元素，此值将被视为 `start` 。
```
Comment (yin1999) [inline files/zh-cn/web/css/justify-content/index.md:60]:
```suggestion
  - : 元素紧密地排列在弹性容器的主轴起始侧。仅应用于弹性布局的项目。对于不是弹性容器里的元素，此值将被视为 `start`。
```

File: files/zh-cn/web/css/justify-content/index.md [hunk 20]
```diff
@@ -42,46 +50,44 @@ justify-content: initial;
...
 - `flex-end`
-  - : 从行尾开始排列。每行最后一个弹性元素与行尾对齐，其他元素将与后一个对齐。
+  - : 元素紧密地排列在弹性容器的主轴结束边缘。仅应用于弹性布局的元素。对于不是弹性容器里的元素，此值将被视为 `end` 。
```
Comment (yin1999) [inline files/zh-cn/web/css/justify-content/index.md:62]:
```suggestion
  - : 元素紧密地排列在弹性容器的主轴结束侧。仅应用于弹性布局的元素。对于不是弹性容器里的元素，此值将被视为 `end`。
```

File: files/zh-cn/web/css/justify-content/index.md [hunk 21]
```diff
@@ -42,46 +50,44 @@ justify-content: initial;
...
 - `space-between`
   - : 在每行上均匀分配弹性元素。相邻元素间距离相同。每行第一个元素与行首对齐，每行最后一个元素与行尾对齐。
 - `space-around`
   - : 在每行上均匀分配弹性元素。相邻元素间距离相同。每行第一个元素到行首的距离和每行最后一个元素到行尾的距离将会是相邻元素之间距离的一半。
 - `space-evenly`
   - : flex 项都沿着主轴均匀分布在指定的对齐容器中。相邻 flex 项之间的间距，主轴起始位置到第一个 flex 项的间距，主轴结束位置到最后一个 flex 项的间距，都完全一样。
 - `stretch`
-  - : If the combined size of the items is less than the size of the alignment container, any `auto`-sized items have their size increased equally (not proportionally), while still respecting the constraints imposed by {{cssxref("max-height")}}/{{cssxref("max-width")}} (or equivalent functionality), so that the combined size exactly fills the alignment container along the main axis.
+
+  - : 如果元素沿主轴的组合尺寸小于对齐容器的尺寸，任何 auto-sized 元素都会等比例地增加其尺寸（而不是按比例增加），同时仍然遵守由 {{cssxref("max-height")}}/{{cssxref("max-width")}}（或相应功能）施加的约束, 以便沿主轴完全填充对齐容器的组合尺寸。
```
Comment (yin1999) [inline files/zh-cn/web/css/justify-content/index.md:77]:
```suggestion
  - : 如果元素沿主轴的组合尺寸小于对齐容器的尺寸，任何尺寸设置为 `auto` 的元素都会等比例地增加其尺寸（而不是按比例增加），同时仍然遵守由 {{cssxref("max-height")}}/{{cssxref("max-width")}}（或相应功能）施加的约束, 以便沿主轴完全填充对齐容器的组合尺寸。
```

File: files/zh-cn/web/css/justify-content/index.md [hunk 22]
```diff
@@ -42,46 +50,44 @@ justify-content: initial;
...
+
+    > **备注：** 虽然在弹性盒子支持 `stretch` 属性，但当应用于弹性盒子时，由于拉伸是由 {{CSSXref("flex")}} 属性控制的，所以 `stretch` 的行为与 `start` 相同。
```
Comment (yin1999) [inline files/zh-cn/web/css/justify-content/index.md:79]:
```suggestion
    > **备注：** 虽然弹性盒子支持 `stretch` 属性，但将其应用于弹性盒子时，由于拉伸是由 {{CSSXref("flex")}} 属性控制的，所以 `stretch` 的行为与 `start` 相同。
```

File: files/zh-cn/web/css/justify-content/index.md [hunk 23]
```diff
@@ -42,46 +50,44 @@ justify-content: initial;
...
+
 - `safe`
-  - : 与对齐关键字一起使用，如果选定的关键字会导致元素溢出容器造成数据丢失，那么将会使用 `start` 代替它。
+  - : 如果元素溢出对齐容器，则元素将按照对齐模式为 `start` 进行对齐。所期望的对齐将不会被实现。
 - `unsafe`
-  - : Regardless of the relative sizes of the item and alignment container, the given alignment value is honored.
+  - : 即使元素溢出对齐容器，也会实现所需的对齐方式。与 `safe` 不同，`safe` 会忽略所需的对齐方式以防止溢出。
```
Comment (yin1999) [inline files/zh-cn/web/css/justify-content/index.md:84]:
```suggestion
  - : 即使元素溢出对齐容器，也会实现所需的对齐方式。与 `safe` 不同，`safe` 会忽略所要求的对齐方式以防止溢出。
```

**Review by yin1999 — APPROVED** [review]
Thanks

**Pull request merged**